src/main/resources/Locators.properties

```diff
@@ -0,0 +1,5 @@
+url=http://localhost:8080
+signed.in.successfully=Signed in successfully
+time.wait.locator=15
+default.user.name=default
+default.user.password=default23
```

Comment:
it's a security violation to store passwords in property files. you can store them as env variables

Reply:
Thanks for your update but it's a test user. It isn't important for local env for now.